build.gradle

```diff
@@ -89,3 +89,7 @@ dependencies {
 tasks.withType(JavaCompile) {
 	options.encoding = 'UTF-8'
 }
+
+tasks.withType(ScalaCompile).configureEach {
+	scalaCompileOptions.additionalParameters = ["-print-tasty"]
+}
```

Comment:
This doesn't seem necessary to me. Scala sources are included just fine without this addition, but just with the fix in mavenCentralPublish.gradle